docker/Dockerfile

```diff
@@ -50,6 +50,6 @@ RUN apt-get update && \
 RUN pip3 install base58
 
 # Install avalanche-cli
-RUN curl -sSfL https://raw.githubusercontent.com/ava-labs/avalanche-cli/main/scripts/install.sh | sh -s -- -b /usr/local/bin v1.3.7
```

Comment:
Any issues running the docker images with the upgraded CLI version?

Reply:
Confirm it works with the new version now (needed to add a couple flags to make the command non-interactive).

Long term, we'll remove the Dockerfile set up all together in favor of the embedded Teleporter + AWM relayer support in the AvalancheCLI 🙌